django/http/response.py

```diff
@@ -17,73 +17,10 @@
 from django.utils.encoding import force_bytes, force_text, force_str, iri_to_uri
 from django.utils.http import cookie_date
 from django.utils.six.moves import map
+from django.utils.six.moves.http_client import responses
```

Comment:
from django.utils.six.moves.http_client import responses as REASON_PHRASES

And then keep using REASON_PHRASES below. That will give a slight shim to anyone who imports REASON_PHRASES.

We could then also leave the STATUS_CODE_TEXT shim untouched.